version-osticket-plugins.env

```diff
@@ -0,0 +1,2 @@
+OSTICKET_PLUGINS_VERSION=93d7d6d11670c7eac7a4e432dbc15f40375a70cf
+OSTICKET_PLUGINS_SHA256SUM=0d4b60045be607d377a7d27a3d5143d5db36041f992c8924816604a81bb342d6
```

Comment:
I don't see this file being referenced outside the Dockerfile, so what's the point?